topics/genome-annotation/tutorials/functional/tutorial.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: tutorial_hands_on
+
+title: Functional annotation of protein sequences
+zenodo_link: https://zenodo.org/record/6861851
+tags:
+  - eukaryote
+  - prokaryote
+questions:
+  - How to perform functional annotation on protein sequences?
+objectives:
+  - Perform functional annotation using EggNOG-mapper and InterProScan
+time_estimation: 1h
+level: Introductory
+key_points:
+  - EggNOG Mapper compares sequences to a database of annotated orthologous sequences
+  - InterProScan detects known motifs in protein sequences
+contributors:
+  - abretaud
+  - erasmusplus
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.